src/metatensor/models/cli/train_model.py

```diff
@@ -192,70 +178,68 @@ def _train_model_hydra(options: DictConfig) -> None:
     )
     train_targets = read_targets(train_options["targets"])
     train_dataset = Dataset(train_structures, train_targets)
+    train_size = 1.0
```

Comment:
Is this hardcoded now?

Reply:
No, if you read further below thsi is adjusted.